apps/tx-builder/public/manifest.json

```diff
@@ -8,5 +8,6 @@
       "sizes": "256x256",
       "type": "image/png"
     }
-  ]
+  ],
+  "safe_apps_permissions": ["clipboard-write"]
```

Comment:
Shouldn't it be camelCase? https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions

Reply:
We are using here a custom property different than the standard one as we want the different directives allowed in the Feature-Policy API and we just translate them to the allow attribute in the iframe.

The permissions standard one i think allow another subset and have implications in the browser behavior